packages/ove-core/package.json

```
@@ -22,6 +22,7 @@
     "express-ws": "4.0.0",
     "http-status-codes": "1.3.0",
     "request": "2.88.0",
+    "source-map": "^0.7.3",
```

Renovate does not like '^' as it manages the versions.

```suggestion
    "source-map": "0.7.3",
```